core/src/main/java/org/apache/accumulo/core/conf/ConfigurationTypeHelper.java

```diff
@@ -20,15 +20,15 @@
 
 import static java.util.concurrent.TimeUnit.DAYS;
 import static java.util.concurrent.TimeUnit.HOURS;
-import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 import java.io.IOException;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
```

Comment:
Instead of doing a non-static import and then modifying so many lines below, you should just update the (now unused) static imports to use ChronoUnit instead of TimeUnit.

Reply:
Good idea. Did this in b016ba9d11